src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/ShareSamplesPage.java

```diff
@@ -11,7 +11,7 @@
 import static ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage.waitForTime;
 
 public class ShareSamplesPage {
-	@FindBy(css = ".t-share-project .ant-select-selection-search-input")
+	@FindBy(css = ".t-project-select .ant-select-selection-search-input")
 	private WebElement shareProjectSelect;
 
 	@FindBy(className = "ant-select-dropdown")
```

Comment:
Can you add testing for searching for a project by it's identifier please

Reply:
Please see 3aa96c6597b5079a30abc048e7285d72f70ca7a5.